### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AzarkontV07**
{
	"$schema": "https://schemas.premid.app/metadata/1.10",
	"author": {
		"name": "Trinityseven0 and EGGSY",
		"id": "162969778699501569"
	},
	"service": "DiziBOX",
	"description": {
		"en": "DiziBOX is one of the biggest Turkish foreign TV show providers on internet.",
		"tr": "DiziBOX, son çıkan, güncel ve bir çok yabancı diziyi izleyebileceğiniz bir yabancı dizi sitesidir.",
		"nl": "DiziBOX is een van de grootste aanbieders van Turkse buitenlandse TV-programma's op internet."
	},
	"url": [
		"www.dizibox.de",
		"diziboxx.com"
	],
	"version": "1.1.27",
	"logo": "https://cdn.rcd.gg/PreMiD/websites/D/DiziBOX/assets/logo.png",
	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/D/DiziBOX/assets/thumbnail.png",
	"color": "#212426",
	"category": "videos",
	"tags": [
		"video",
		"tvshows",
		"turkish"
	],
	"iframe": true,
	"iFrameRegExp": ".*"
}

### CODE REVIEW

**Comment by DarkVillager** [comment]
> { "$schema": "https://schemas.premid.app/metadata/1.10", "author": { "name": "Trinityseven0 and EGGSY", "id": "162969778699501569" }, "service": "DiziBOX", "description": { "en": "DiziBOX is one of the biggest Turkish foreign TV show providers on internet.", "tr": "DiziBOX, son çıkan, güncel ve bir çok yabancı diziyi izleyebileceğiniz bir yabancı dizi sitesidir.", "nl": "DiziBOX is een van de grootste aanbieders van Turkse buitenlandse TV-programma's op internet." }, "url": [ "www.dizibox.de", "diziboxx.com" ], "version": "1.1.27", "logo": "https://cdn.rcd.gg/PreMiD/websites/D/DiziBOX/assets/logo.png", "thumbnail": "https://cdn.rcd.gg/PreMiD/websites/D/DiziBOX/assets/thumbnail.png", "color": "#212426", "category": "videos", "tags": [ "video", "tvshows", "turkish" ], "iframe": true, "iFrameRegExp": ".*" }

Use the format that's there (aka the default) when creating a pr.

**Comment by DarkVillager** [comment]
what

**Comment by DarkVillager** [comment]
this already exists? [https://github.com/PreMiD/Presences/tree/main/websites/D/DiziBOX](https://github.com/PreMiD/Presences/tree/main/websites/D/DiziBOX)

File: websites/iframe.ts
Comment (DarkVillager) [inline websites/iframe.ts:1]: Wrong location see comment https://github.com/PreMiD/Presences/pull/8643#issuecomment-2267514848

File: websites/metadata.json
Comment (DarkVillager) [inline websites/metadata.json:1]: Wrong location see comment https://github.com/PreMiD/Presences/pull/8643#issuecomment-2267514848

File: websites/presence.ts
Comment (DarkVillager) [inline websites/presence.ts:1]: Wrong location see comment https://github.com/PreMiD/Presences/pull/8643#issuecomment-2267514848

**Comment by theusaf** [comment]
This PR doesn't change anything in the code besides the author's name. If there are no actual changes, the PR will be closed.

**Pull request closed without merging**